templates/hugo/files/.platform.app.yaml

```diff
@@ -0,0 +1,26 @@
+# .platform.app.yaml
+
+# The name of this application, which must be unique within a project.
+name: 'hugo'
+
+# The type key specifies the language and version for your application.
+type: 'golang:1.12'
+
+# The hooks that will be triggered when the package is deployed.
+hooks:
+    # Build hooks can modify the application files on disk but not access any services like databases.
+    build: |
+      ./build.sh
```

Comment:
We can go ahead and use the !include syntax here.  It should work for plain text still, even if archive is broken.